### ISSUE TITLE
Nothing happen using python 3.8

### ISSUE THREAD

**Issue by Adelax92** [UxxHans/Rainbow_Cats_Universal_FPS_Aimbot_Experiment_Only#1]
What version of Python are u using? Using 3.8 trying to open nothing 
![capitan-america-lanzando-escudo_1920x1080_xtrafondos com](https://user-images.githubusercontent.com/92597817/144506834-1da25ae2-d277-4b3c-add8-9670d6ce2527.jpg)
happen.

**Comment by UxxHans** [comment]
Yes, it won't work. You will only see the basic modules of codes, I mentioned that in order to anti-cheat, I didn't release the main part of the code.

**Comment by UxxHans** [comment]
Nice Captain American picture btw

**Comment by UxxHans** [comment]
If you want to experiment, I can sent it to you.

**Comment by Adelax92** [comment]
Of course ! how can u send me?

**Comment by Adelax92** [comment]
i was testing so much with yolo and could be good test it could help to improve and test!

**Comment by Adelax92** [comment]
adelax1992@gmail.com if u can send it.

**Comment by DragoHG** [comment]
Can you send for me either? fabio.monreal@outlook.com

Thanks :D

**Comment by dankovix** [comment]
I would like to receive main.py you can send me in my email to test this project
( lucasvacilosantos@gmail.com )

**Comment by UxxHans** [comment]
Sorry guys, I don't know if sending it is legal or not. It's hard to decide if I should release the code. I'll keep the code private for the moment.
